### PR TITLE
New visualization package update

### PR DESCRIPTION
**Pull request by dheeraj2data**
Added about "facets", A project under "People+AI Research (PAIR) Initiative"

### CODE REVIEW

**Comment by jbednar** [comment]
Thanks for the PR! I have not seen this project before, but from the [github repo](https://github.com/PAIR-code/facets) and [website](https://pair-code.github.io/facets/) I'm not able to be sure that it's really a Python visualization library; most of the focus seems to be on the HTML objects it provides, though there does seem to be some Python in the repo. Assuming it can indeed be considered a Python viz library (the focus of PyViz.org), then because it says it provides "Visualizations for machine learning datasets", it should go into the "Other domain-specific" category, where other ML viz libraries like yellowbrick are listed. If (a) it's indeed a Python viz library, and (b) it's moved to the domain-specific category, I'm happy to merge it!

**Comment by dheeraj2data** [comment]
Sure @jbednar .Let's will put it under "domain-specific category", Shall I create new PR?

**Comment by jbednar** [comment]
You can use this PR; just move that bit to the end of the other section and I'll squash-merge so that the earlier location won't be in the history anyway.

File: tools/tools.yml [hunk 1]
```diff
@@ -154,6 +154,10 @@
     - repo: biggles-plot/biggles
       site: https://biggles-plot.github.io
       badges: pypi
+      
+    - repo: PAIR-code/facets
+      site: https://pair-code.github.io/facets/
+      badges: pypi
```
Comment (jbednar) [inline tools/tools.yml:160]:
```suggestion
```

**Pull request merged**